cypress/features/regression/test/actionPopoverIFrameNoIFrame.feature

```diff
@@ -79,6 +102,20 @@ Feature: Action Popover component
     When I press ESC on focused element
     Then Action Popover element is not visible
 
+  @positive
+  Scenario: Close ActionPopover using ESC key
```

Comment:
you should also check that focused element hasn't submenu
```suggestion
  Scenario: Close ActionPopover using ESC key if it hasn't submenu
```

Reply:
updated